docs/user-documentation/track-upload.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Track Upload Status
+id: track-upload
+slug: /track-upload
+sidebar_label: Track Upload
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+In Swarm, an instruction to upload data to the network goes through 3 consecutive stages before it is completed:
+
+- Splitting
+- Storing
+- Sending
+
+In the splitting state, the file is deconstructed in chunks (Swarms canonical data unit) and packaged in a [Binary Merkle Tree](https://en.wikipedia.org/wiki/Merkle_tree). After splitting, the chunks are stored in your local database where they enter a queue, to be sent to the network.
+
+Sending starts immediately when the first chunks are split and stored. After the chunk is sent, your node will receive a receipt from the node that has stored the chunk, marking the completion of the upload for that chunk. After a receipt has been received for all chunks, the upload is complete.
+
+## What is a tag
+The status of your upload can be followed by using `tags`. A `tag` is a label given to all chunks that belong to the same upload instruction. In the `bee-js` library, tag looks as follows:
+
+```ts
+interface Tag {
```

Comment:
Just a heads-up that the `Tag` definition is about to change:
https://github.com/ethersphere/bee-js/blob/a266f551ce89b3e46dc315ee619f85b2df694e69/src/types/index.ts#L32